src/components/usa-tag/usa-tag.stories.js

```diff
@@ -0,0 +1,21 @@
+import "../../stylesheets/uswds.scss";
+import Component from "./usa-tag.twig";
+import Data from "./usa-tag.json";
+import bigData from "./usa-tag~big.json";
+
+export default {
+  title: "Components/Tags",
+  argTypes: {
+    text: {
```

Comment:
Do we need this "text" argType?

Reply:
I switched it to title so users can modify the tag text if they wanted to play with it. Should I also get rid of the modifier option components that don't use one (like info tag) or leave it for people to add modifiers for testing?

![image](https://user-images.githubusercontent.com/25211650/151201776-33b7a972-9eae-488a-af67-29c0600b4b8d.png)